program/src/state.rs

```diff
@@ -27,6 +27,11 @@ use crate::{
 };
 use crate::{REWARDS_WITHDRAW_AUTHORITY, VALIDATOR_STAKE_ACCOUNT, VALIDATOR_UNSTAKE_ACCOUNT};
 
+// Nit:
+// Since an account is initialized with 0'ed out data, you can't tell if an
+// account has been initialized or not if you're looking for a 0.  That's fine
+// because you look through all the bytes on init, but for simplicity, you could
+// make this `1` and only check for that on init.
```

Comment:
We’ll go with the 0 for now because other projects already started integrating.

Reply:
Fair enough, it's not actually a problem, just something to keep in mind for next time